src/nvim/main.c

```diff
@@ -2115,3 +2093,27 @@ static void check_swap_exists_action(void)
     getout(1);
   handle_swap_exists(NULL);
 }
+
+/// Sets up the terminal window for use.
+///
+/// This must be done after resetting full_screen, otherwise it may move the
+/// cursor.
+///
+/// @remark We may call mch_exit() before calling this.
+static void term_init(void)
+{
+  Columns = 80;
+  Rows = 24;
```

Comment:
Why set these two variables? I ask because nvim on this branch seems to work fine in a tmux pane smaller than these dimensions, and when I remove these two lines all tests still pass and everything still seems to work fine.